### PR TITLE
Move documentation from the wiki into the code base docs directory

### PR DESCRIPTION
**Pull request by xamanu**

### CODE REVIEW

**Comment by xamanu** [comment]
There are some pages of the wiki, that I didn't move into this PR:

* Install scripts are going to be covered in #1345
* [Ideas on Generation Session Notes](https://github.com/hotosm/tasking-manager/wiki/Onboarding-Idea-Generation-Session-Notes), which I'd suggest to move into an issue.
* [TM3-Production-with-Nginx-Setup-Guide](https://github.com/hotosm/tasking-manager/wiki/TM3-Production-with-Nginx-Setup-Guide), dropped because of being outdated and not used by us anymore.
* [ML4TM](https://github.com/hotosm/tasking-manager/wiki/ML4TM) is deprecated
* [Dev-Ops](https://github.com/hotosm/tasking-manager/wiki/Dev-Ops) outdated, needs to be rewritten entirely.
* [Docker Integration](https://github.com/hotosm/tasking-manager/wiki/Docker-Integration), outdated and almost no information, needs to be written.
* [TM3 Wireframes](https://github.com/hotosm/tasking-manager/wiki/Initial-TM3-Wireframes), outdated and probably not the right place.

Further I combined:

* Development setup
  * Readme information on dev-setup
  * [Test API locally](https://github.com/hotosm/tasking-manager/wiki/Testing-API-locally-via-swagger-with-an-auth-token)
* Frontend development
  * [Restricting features to certain users](https://github.com/hotosm/tasking-manager/wiki/Restricting-features-to-certain-users)

File: README.md [hunk 1]
```diff
@@ -29,159 +29,26 @@ Afterwards copy the consumer key and secret from OpenStreetMap into your configu
...
+* Review our [Contributor guidelines](./docs/contributing.md)
+* And help us to [translate the TM](./docs/localise.md)
+* Come, attend the [working group meetings](./docs/working-group.md)
+* Make sure you are aware of our [Code of Conduct](./docs/code_of_conduct.md)
 
-## Wiki
+## Developers, developers, developers
 
-In addition to this README, you might want to check the [Tasking Manager Wiki](https://github.com/hotosm/tasking-manager/wiki)
+* See how you can [costumize your instance of the TM](./docs/costumize.md)
```
Comment (dakotabenjamin) [inline README.md]: Just a small typo here: customize

Comment (xamanu) [reply README.md]: Thanks for noticing. Fixed.

File: README.md [hunk 2]
```diff
@@ -29,159 +29,26 @@ Afterwards copy the consumer key and secret from OpenStreetMap into your configu
...
 
 * One command to get everything together and start the Tasking Manager: `docker-compose up -d`
 * Visit with your browser `http://127.0.0.1:5000`
 
-
 For stopping this command do the job: `docker-compose stop`
 And you can check the logs with `docker-compose logs -f`
 
-Alternatively you can install a [development setup](./README.md#development-setup).
+**Alternatively** you can review how to install a [development setup](./docs/setup-development.md) or deploy a [live installation](./docs/setup-live.md).
```
Comment (willemarcel) [inline README.md]: It's missing the `setup-live.md` file

File: docs/migration-tm2-to-tm3.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* You have [installed python3.6 (preferably in a virtual environment) and all of the requirements via pip](https://github.com/hotosm/tasking-manager#dependencies).
+* You have [installed all of the nodeJS modules in the client folder and gulp built the client](https://github.com/hotosm/tasking-manager#client-development).
+* You have [set all the environmental variables needed to run the TM3](https://github.com/hotosm/tasking-manager#environment-vars).
+* You have [created a new database for TM3](https://github.com/hotosm/tasking-manager/wiki/Dev-Ops) with the same owner as your TM2 database.
+* You have [populated the database schema for TM3 using alembic](https://github.com/hotosm/tasking-manager#creating-the-db).
+
+A check to ensure all of these conditions are met is to start the TM3 app, launch the main homepage , and click on while watching the console. It will say there are currently no projects, which is expected since you have a skeleton database, but you should make sure no errors are thrown when the application tries checking for projects. If the page loads successfully and no errors occur, then you should be set!
+
+### Installation Example Walkthroughs
```
Comment (willemarcel) [inline docs/migration-tm2-to-tm3.md]: There are some links to the wiki in that section. Should we move the linked content to the `docs` folder or just link to the `setup-development.md` file?

File: README.md (hunk 2 above)
Comment (xamanu) [reply README.md]: Thanks for noticing. Removed for now, as we deprecated [TM3-Production-with-Nginx-Setup-Guide](https://github.com/hotosm/tasking-manager/wiki/TM3-Production-with-Nginx-Setup-Guide).

File: docs/migration-tm2-to-tm3.md (hunk 3 above)
Comment (xamanu) [reply docs/migration-tm2-to-tm3.md]: Right. I put a link to the dev environment setup and rephrased the following sentence a bit.

File: docs/setup-development.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+4.  Confirm the image is running successfully:
+
+`
+docker ps
+`
+
+5. Finally you can set your env variable to point at your containerised DB:
+
+`
+export TM_DB=postgresql://hottm:hottm@localhost/tasking-manager
```
Comment (willemarcel) [inline docs/setup-development.md]: I think that TM_DB variable is outdated. On the top of the page it recommends another set of env vars.

File: docs/setup-development.md [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+$ CREATE USER "hottm" PASSWORD 'hottm';
+$ CREATE DATABASE "tasking-manager" OWNER "hottm";
+$ \c "tasking-manager";
+$ CREATE EXTENSION postgis;
+```
+
+Finally, add the environmental variable to access the database:
+
+`
+export TM_DB=postgresql://hottm:hottm@localhost/tasking-manager
```
Comment (willemarcel) [inline docs/setup-development.md]: I think that TM_DB variable is outdated. On the top of the page it recommends another set of env vars.

File: docs/setup-development.md (hunk 4 above)
Comment (xamanu) [reply docs/setup-development.md]: `TM_DB` still works, but the other ones should be mentioned here. However this PR is more about moving the pages in. I agree with you that we have to go through the whole documentation and update it. This PR can only be a first step.

File: docs/setup-development.md (hunk 5 above)
Comment (xamanu) [reply docs/setup-development.md]: `TM_DB` still works, but the other ones should be mentioned here. However this PR is more about moving the pages in. I agree with you that we have to go through the whole documentation and update it. This PR can only be a first step.

File: docs/running-tests.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Run tests
```
Comment (willemarcel) [inline docs/running-tests.md]: I think we could delete this file, as we have instruction about how to run tests in the `setup-development.md`

**Pull request merged**